tests/vmi_slirp_interface_test.go

```diff
@@ -140,13 +142,15 @@ var _ = Describe("[Serial]Slirp Networking", func() {
 		log.Log.Infof("%v", output)
 		Expect(err).To(HaveOccurred())
 
-		By("communicate with the outside world")
-		Expect(console.SafeExpectBatch(vmi, []expect.Batcher{
-			&expect.BSnd{S: "\n"},
-			&expect.BExp{R: console.PromptExpression},
-			&expect.BSnd{S: "curl -o /dev/null -s -w \"%{http_code}\\n\" -k https://google.com\n"},
-			&expect.BExp{R: "301"},
-		}, 180)).To(Succeed())
+		if len(flags.ConnectivityCheckDNS) != 0 {
+			By("communicate with the outside world")
+			Expect(console.SafeExpectBatch(vmi, []expect.Batcher{
+				&expect.BSnd{S: "\n"},
+				&expect.BExp{R: console.PromptExpression},
+				&expect.BSnd{S: fmt.Sprintf("curl -o /dev/null -s -w \"%%{http_code}\\n\" -k https://%s\n", flags.ConnectivityCheckDNS)},
```

Comment:
@alonSadan `w \"%{http_code}\\n\` -> `-w \"%%{http_code}\\n\` you have added additional `%` is that in purpose?

Reply:
Because when I used `fmt.Sprintf` I had to escape the `%`. the escape char for `%` is `%`.